packages/components/Shimmer/ios/ShimmerContainerView.swift

```diff
@@ -0,0 +1,144 @@
+import Foundation
+import FluentUI
+
+
+@objc(MSFShimmerContainerView)
+class ShimmerContainerView: UIView {
+
+	@objc var appearance: NSDictionary = NSDictionary() {
```

Comment:
each property should have documentation

Reply:
Nit: I don't know if appearance and shimmerAppearance variable names are clear. maybe it will help with more comments. but different updateshimmerViewappearnce and updateshimmerappearance needs more clarity.

Reply:
Even if they are documented in the ShimmerView itself, and on the react native side? I just didn't document them in this containerView thing which to me is an implementation detail

Reply:
I'll prefer to overdocument over underdocument, and add more comments here